craft_parts/lifecycle_manager.py

```diff
@@ -103,7 +104,44 @@ def plan(self, target_step: Step, part_names: Sequence[str] = None) -> List[Acti
         :return: The list of :class:`Action` objects that should be executed in
             order to reach the target step for the specified parts.
         """
-        # TODO: verify if base packages changed
-
         actions = self._sequencer.plan(target_step, part_names)
         return actions
+
+
+def _build_part(name: str, spec: Dict[str, Any], project_dirs: ProjectDirs) -> Part:
+    if not isinstance(spec, dict):
+        raise errors.PartSpecificationError(
+            part_name=name, message="part definition is malformed"
+        )
+
+    plugin_name = spec.get("plugin", "")
+
+    fallback_plugin_name = not plugin_name
```

Comment:
```suggestion
    part_name_as_plugin_name = not plugin_name
```

Reply:
Renamed variable, and added a comment as well.